pkg/controller/netflow.go

```diff
@@ -80,15 +80,16 @@ func (cont *AciController) initNetflowInformerBase(listWatch *cache.ListWatch) {
 
 }
 
-func (cont *AciController) netflowPolicyUpdated(obj interface{}) {
+func (cont *AciController) netflowPolicyUpdated(obj interface{}) bool {
 	netflowPolicy := obj.(*netflowpolicy.NetflowPolicy)
 	key, err := cache.MetaNamespaceKeyFunc(netflowPolicy)
 	if err != nil {
 		NetflowPolicyLogger(cont.log, netflowPolicy).
 			Error("Could not create key:" + err.Error())
-		return
+		return false
 	}
 	cont.queueNetflowUpdateByKey(key)
+	return false
```

Comment:
Shouldn't this return true? Right now it always returns false...

Reply:
You are right! I have addressed your comments in the commit below.

Reply:
you don't need a return bool value here.  make it void function you are not handling any error condition here.

Reply:
changed it.